picard/const/__init__.py

```diff
@@ -68,20 +69,27 @@
 CAA_HOST = "coverartarchive.org"
 CAA_PORT = 443
 
+# Prepare documentation URLs
+if PICARD_VERSION.identifier == 'final':
+    DOCS_VERSION = "v{}".format(PICARD_VERSION.to_string(short=True))
+else:
+    DOCS_VERSION = ""  # points to latest version
+DOCS_BASE_URL = "https://picard-docs.musicbrainz.org/" + DOCS_VERSION + '/en'
```

Comment:
For latest version, this will set `DOCS_BAS_URL` to "https://picard-docs.musicbrainz.org//en" (note the double slash).  Not sure if this is a problem.  Perhaps something like:

```
if PICARD_VERSION.identifier == 'final':
    DOCS_VERSION = "v{}/".format(PICARD_VERSION.to_string(short=True))
else:
    DOCS_VERSION = ""  # points to latest version
DOCS_BASE_URL = "https://picard-docs.musicbrainz.org/" + DOCS_VERSION + 'en'
```

Reply:
Yes, I moved the language handling out of the options to const and this is the result. I fixed this.